### DISCUSSION TITLE
v9 KubeOps KubernetesClient doesn't expose APIClient

### DISCUSSION THREAD

**Discussion by mahavirvemuri** [dotnet/dotnet-operator-sdk#757] [Q&A] [answered]
Hi,  I am upgrading KubeOps v7.6.1 to KubeOps.Operator. The KubernetesClient v9 doesn't expose the ApiClient.

This PR [749](https://github.com/buehler/dotnet-operator-sdk/pull/749) is currently in the pre-release v9-pre.4. Would you happen to know when can we expect a new stable version that includes the above?

**Comment by buehler** [comment, marked as answer]
It is released now, I messed up with the tag config of semantic release

**Reply by mahavirvemuri** [reply]
Thank you :) The version v9.0.2 has it all now.
